nvdaHelper/readme.md

```diff
@@ -71,3 +71,15 @@ When `fillVBuf` is called, it recursively descends through the child elements th
 `controlFieldNode`s or `textFieldNode`s for them.
 This code is responsible for interacting with the IA2 accessibility API, these calls should be minimised for
  performance reasons.
+
+
+### Overview
+
+- NVDA's python code calls into the (`NVDAHelperLocal.dll` / `NVDAHelperLocalWin10.dll`).
+- Generated RPC Wrappers are called.

```

Comment:
client-side RPC wrappers are only available in nvdaHelperLocal.dll. The above text seems to suggest that nvdaHelperLocalWin10.dll also contains them which is not correct.

Reply:
Thanks, I'll try to clarify this further.